### PR TITLE
0.5 cleanup spec added to json tests

### PR DESCRIPTION
**Pull request by eupakhomov**

### CODE REVIEW

**Comment by TVolden** [comment]
The CI terminates in a new and weird way for this PR:
```
[...]
12:13:35.405 [main] DEBUG eu.chargetime.ocpp.JSONClient - Feature repository: FeatureRepository{featureList.size=21}
12:13:35.413 [WebSocketWorker-226] DEBUG eu.chargetime.ocpp.WebSocketListener - On connection open (resource descriptor: /testdummy)
12:13:35.413 [WebSocketWorker-226] DEBUG eu.chargetime.ocpp.Server - Session created: 869b6904-0f3a-4172-b438-600dcc9d3498
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```

**Comment by eupakhomov** [comment]
Seems build hangs for some reason although nothing was added but cleanup (server side stop). I will think how to reproduce locally.

**Comment by TVolden** [comment]
I'm sure you have seen, but the IC says 16 failures and 6 errors now:
```
[ERROR] Tests run: 43, Failures: 14, Errors: 6, Skipped: 0
```

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/WebSocketListener.java [hunk 1]
```diff
@@ -132,6 +132,9 @@ public void onStart() {
         }
 
         server.setConnectionLostTimeout(pingInterval);
+
+        server.setReuseAddr(true);
```
Comment (sumlin) [inline ocpp-v1_6/src/main/java/eu/chargetime/ocpp/WebSocketListener.java]: That's what might solve all 'not connected' errors.

Comment (sumlin) [reply ocpp-v1_6/src/main/java/eu/chargetime/ocpp/WebSocketListener.java]: @eupakhomov well, sometimes if you stops your bind-address operation, Linux core doesn't release a port, it is not according WS, that's core network stuff. I have no idea why 'false' state is a default state, so I vote to set is as 'true' by default, I do this all my life without any problems. @TVolden what do you think?

**Comment by eupakhomov** [comment]
@sumlin do you have also client-side related experience? If mentioned above
is truth for Android -based devices as well?

чт, 19 Апр 2018 г., 10:56 sumlin <notifications@github.com>:

**Comment by eupakhomov** [comment]
@TVolden @sumlin so `reuseAddr` by default set to `true` for the server.

**Comment by TVolden** [comment]
Yeah let's just have it set to true by default, that sounds good. Can't wait to stabilize the intrgration tests, they kinda loose their value when they fail all the time.

I'll review it tomorrow. Thanks for all your hard work, I really appreciate it.

**Comment by TVolden** [comment]
Looks like a small merge conflict with the imports

**Pull request merged**